Godeps/Godeps.json

```diff
@@ -10,6 +10,10 @@
 			"Rev": "70b2c90b260171e829f1ebd7c17f600c11858dbe"
 		},
 		{
+			"ImportPath": "github.com/microsoft/hcsshim",
+			"Rev": "fc8f843b468326d898f2ef2257440c792d2fa283"
+		},
+		{
```

Comment:
Also can you import using vendor.sh - docker master currently isn't using Godeps.  (Ping me internally about this)